src/components/sandbox/components/VmDetails.tsx

```diff
@@ -98,6 +97,7 @@ const VmDetails: React.FC<VmDetailsProps> = ({
     const [outboundRuleChanged, setOutboundRuleChanged] = useState<boolean>(false);
     const [saveIsEnabled, setSaveIsEnabled] = useState<ButtonEnabledObj>({ enabled: false, error: '' });
     const dispatch = useDispatch();
+    const resources = useSelector(getResourcesFromStore())
```

Comment:
Forgot semicolon

Reply:
I will fix this when I commit the fixes for testing.